README.md

```diff
@@ -178,6 +178,23 @@ sfdx force:mdapi:deploy -d destructiveChanges --ignorewarnings
 
 And voilà! 🥳
 
+### Diff between branches:
+
+SGD works with any git sha pointer: commit sha, branch, tag, git expression (HEAD, etc.).
+
+Here are 2 examples showing how you can compare the content of different branches:
+
+**1) Comparing commits in different branches**
+(for example, if you have commit `fbc3ade6` in branch `develop` and commit `61f235b1` in branch `master`)
+```
+sfdx sgd:source:delta --to fbc3ade6 --from 61f235b1 --output .
+```
+
+**2) Comparing branches**
+```
+sfdx sgd:source:delta --to develop --from master --output .
+```
+
```

Comment:
Should we also add tag example ?

Reply:
I didn't wait to make it too long :-)
What about leaving just those 2 examples for now, and consider adding more examples if we get additional questions?